pkg/rosa/helpers.go

```diff
@@ -6,10 +6,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const hostedCpFlagName = "hosted-cp"
+
 func HostedClusterOnlyFlag(r *Runtime, cmd *cobra.Command, flagName string) {
-	isFlagSet := cmd.Flags().Changed(flagName)
-	if isFlagSet {
-		r.Reporter.Errorf("Setting the `%s` flag is only supported for hosted clusters", flagName)
-		os.Exit(1)
+	if cmd.Flag(hostedCpFlagName) == nil && (cmd.Flag(hostedCpFlagName) != nil && !cmd.Flag(hostedCpFlagName).Changed) {
```

Comment:
How is it going to be both nil and not nil :thinking: ? Or maybe I'm just reading this wrong

Reply:
Ugh, meant `||`